ipc/hypervisor/src/lib.rs

```diff
@@ -121,6 +127,22 @@ impl Hypervisor {
 			worker.poll()
 		}
 	}
+
+	pub fn shutdown(&self, wait_time: Option<std::time::Duration>) {
+		if wait_time.is_some() { std::thread::sleep(wait_time.unwrap()) }
+
+		let mut childs = self.processes.write().unwrap();
+		for (ref mut binary, ref mut child) in childs.iter_mut() {
+			println!("Stopping {}", binary);
```

Comment:
should be `info!`/`trace!` - don't want technical messages appearing to the user.